sqlglot/dialects/postgres.py

```diff
@@ -237,6 +237,29 @@ def _unix_to_time_sql(self: Postgres.Generator, expression: exp.UnixToTime) -> s
     )
 
 
+def _build_levenshtein_less_equal(args: t.List) -> exp.Levenshtein:
+    # Postgres has two signatures for levenshtein_less_equal function, but in both cases 
+    # max_dist is the last argument
+    # levenshtein_less_equal(source, target, ins_cost, del_cost, sub_cost, max_d)
+    # levenshtein_less_equal(source, target, max_d)
+    max_dist = args.pop()
+
+    return exp.Levenshtein(
+        this=seq_get(args, 0),
+        expression=seq_get(args, 1),
+        ins_cost=seq_get(args, 2),
+        del_cost=seq_get(args, 3),
+        sub_cost=seq_get(args, 4),
+        max_dist=max_dist,
+    )
+
+
+def _levenshtein_sql(self: Postgres.Generator, expression: exp.Levenshtein) -> str:
+    name = "LEVENSHTEIN_LESS_EQUAL" if self.sql(expression, "max_dist") else "LEVENSHTEIN"
```

Comment:
Why do we need to generate `max_dist`? Doesn't it suffice to check `expression.args.get("max_dist")`?